bfapi/service/jobs.py

```diff
@@ -208,7 +208,7 @@ def create(
         tide_max_24h=tide_max,
     )
 
-def __make_cli_cmd(
+def __make_algorithm_cli_cmd(
```

Comment:
My understanding is that it effectively makes it a private function - that it prevents the function from being accessed outside of the local area.  (I don't recall if the "local area" refers to the file itself or the namespace or whatever, but that's the basic idea.  I spent more focus/research on it when I was making the decision).  If I'm incorrect on that, then you have my apologies.
It's notably distinct from naming a function `__blah__` though.  I do recall that.